examples/object_detection/ios/RunScripts/download_models.sh

```diff
@@ -0,0 +1,90 @@
+#  2024 The Google AI Edge Authors. All Rights Reserved.
+
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#  =============================================================================
+
+set -ex
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+MOBILENETV1_SSD_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_ssd_mobilenet_v1_1_metadata_2.tflite"
+EFFICIENTDET_LITE0_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_efficientdet_lite0_detection_metadata_1.tflite"
+EFFICIENTDET_LITE1_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_efficientdet_lite1_detection_metadata_1.tflite"
+EFFICIENTDET_LITE2_URL="https://storage.googleapis.com/download.tensorflow.org/models/tflite/task_library/object_detection/ios/lite-model_efficientdet_lite2_detection_metadata_1.tflite"
+
+MOBILENETV1_SSD_NAME="ssd_mobilenet_v1.tflite"
```

Comment:
Comment probably got lost from when you closed the PR on the other repo - if we aren't using all of these models, please don't download them.